### DISCUSSION TITLE
Test

### DISCUSSION THREAD

**Discussion by kiminuo** [dotnet/maui#1763] [Q&A]

**Comment by vhugogarcia** [comment]
I believe XCT will be still our plugin to use for validations. I have been using it and I like it. However, XCT does not integrate well with MVVM pattern.

**Comment by nogginbox** [comment]
This questions needs some love. I've been looking for some really simple forms based examples. Neither Weather 24 or the podcast app seem to have any entry controls in them.
I'd love a simple example with some data validation. So far all my Bingle searches have come back blank.
